graviti/dataframe/frame.py

```diff
@@ -377,19 +377,19 @@ def shape(self) -> Tuple[int, int]:
         """
         return (self.__len__(), len(self._column_names))
 
-    @property
-    def size(self) -> int:
-        """Return an int representing the number of elements in this object.
-
-        Return:
-            Size of the DataFrame.
-
-        Examples:
-            >>> df = DataFrame({"col1": [1, 2], "col2": [3, 4]})
-            >>> df.size
-            4
-
-        """
+    # @property
+    # def size(self) -> int:
+    #     """Return an int representing the number of elements in this object.
+    #
+    #     Return:
+    #         Size of the DataFrame.
+    #
+    #     Examples:
+    #         >>> df = DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    #         >>> df.size
+    #         4
+    #
+    #     """
```

Comment:
normally we use word `method` to represent the function in class,
so better to use `method` in commit message

Reply:
Also this commit will affect the SDK API, use `refactor` is better.